shell/platform/darwin/macos/framework/Source/FlutterAppLifecycleDelegate.mm

```diff
@@ -101,9 +96,6 @@ - (void)removeDelegate:(NSObject<FlutterAppLifecycleDelegate>*)delegate {
 #define DISTRIBUTE_NOTIFICATION(SELECTOR)                                  \
   -(void)handle##SELECTOR : (NSNotification*)notification {                \
     for (NSObject<FlutterAppLifecycleDelegate> * delegate in _delegates) { \
-      if (!delegate) {                                                     \
```

Comment:
Minor cleanup; this is not doing anything useful since the condition will be false anyway.